.github/workflows/containers.yml

```diff
@@ -18,17 +18,17 @@ jobs:
     PublishJibContainers:
         runs-on: ubuntu-latest
         timeout-minutes: 15
-
         steps:
             -   uses: actions/checkout@v3
                 with:
                     ref: ${{ github.head_ref }}
-
             -   uses: ./.github/actions/setup
-
             -   name: Build with Gradle
                 run: >
                     ./gradlew clean jib -Partifactory_user=${{ secrets.ARTIFACTORY_USERNAME }} -Partifactory_password=${{ secrets.ARTIFACTORY_PASSWORD }} -Pzowe.docker.password=${{ secrets.PERSONAL_JB_TOKEN }} -Pzowe.docker.username=balhar-jakub -Pzowe.docker.container=ghcr.io/balhar-jakub/ -Pzowe.docker.tag=${{ env.JOB_ID }}
+            -   name: Standalone Catalog
+                run: >
+                    ./gradlew :api-catalog-services:jib -Pzowe.jib.image.suffix=standalone -Pzowe.jib.image.javaAgentPort=6305 -Pzowe.jib.image.debugPort=5125 -Partifactory_user=${{ secrets.ARTIFACTORY_USERNAME }} -Partifactory_password=${{ secrets.ARTIFACTORY_PASSWORD }} -Pzowe.docker.password=${{ secrets.PERSONAL_JB_TOKEN }} -Pzowe.docker.username=balhar-jakub -Pzowe.docker.container=ghcr.io/balhar-jakub/ -Pzowe.docker.tag=${{ env.JOB_ID }}
```

Comment:
is this build so different from the rest that it needs to be done separately?

Reply:
This is a way I found to have two images created based on the same project (api-catalog-services). I think the main point to fix is how to customise the location of `api-catalog-services-standalone.yaml` if I use the same image, do you know if there's a way to do it?

Reply:
ok, it seems that JIB is not very good about multiple images from a single gradle file, no need to spend much time on the workaround. thanks for the clarification.